Release 1.2.0 (#121)
audeering/audiofile: CHANGELOG.rst

and this project adheres to `Semantic Versioning`_.

Version 1.2.0 (2023-02-13)
--------------------------

* Added: ``normalize`` argument to ``audiofile.convert_to_wav()``
* Added: ``bit_depth`` argument to ``audiofile.convert_to_wav()``
* Added: ``overwrite`` argument to ``audiofile.convert_to_wav()``
* Added: examples to the API documentation
* Added: example how to use ``audresample.resample()``
  and ``audresample.remix()``
  in the API documentation of ``audiofile.read()``

> * Added: support units and samples
>   and most other formats supported by ``audmath.duration_in_seconds()``
>   for the ``duration`` and ``offset`` arguments
>   of ``audiofile.read()``
>   and ``audiofile.convert_to_wav()``

Reviewer: ```suggestion
* Added: the arguments ``duration`` and ``offset``
  of ``audiofile.read()``
  and ``audiofile.convert_to_wav()``
  can be specified in different units
  or samples and most other formats 
  supported by ``audmath.duration_in_seconds()``
```

> * Added: support for negative values
>   for the ``duration`` and ``offset`` arguments
>   of ``audiofile.read()``
>   and ``audiofile.convert_to_wav()``

Reviewer: ```suggestion
* Added: the arguments ``duration`` and ``offset``
  of ``audiofile.read()``
  and ``audiofile.convert_to_wav()``
  support negative values
```

> * Changed: allow ``audiofile.convert_to_wav()``
>   to be provided by only the input file.
>   The resulting WAV file
>   is then stored under the same path
>   but with ``'wav'`` as file extension

Reviewer: ```suggestion
* Changed: if no output path is provided to
   ``audiofile.convert_to_wav()``,
   it is derived from the input path
   by replacing the file extension with ``'wav'``
```

* Fixed: ensure ``audiofile.convert_to_wav()``
  creates identical result

>   as calling ``audiofile.read()``
>   and ``audiofile.write()``
>   independently

Reviewer: ```suggestion
  as calling ``audiofile.read()``
  followed by ``audiofile.write()``
```

> * Fixed: support for non SND-files

Reviewer: what are "SND-files"?

Author: Files supported by `libsndfile`. But you are right, this is not nice for a changelog. What I mean here is: all files besides WAV, FLAC, OGG

Reviewer: Ok, we should be more verbose here

Author: I changed to

```
* Fixed: support for non WAV, FLAC, OGG files
  like MKA or MP4 
  on Windows network shares
```
